### PR TITLE
fix(challenges): Update stock price checker sample project URL and add proxy info

### LINKED ISSUE #18023: Nasdaq Stock Price Checker Glitch Example not working
The current glitch example  (https://giant-chronometer.glitch.me/) for the [Information Security and Quality Assurance  - Stock Price Checker](https://learn.freecodecamp.org/information-security-and-quality-assurance/information-security-and-quality-assurance-projects/stock-price-checker) project is not currently working.

I have not looked too deeply into the project yet, but this [FCC forum thread](https://forum.freecodecamp.org/t/stock-price-checker-project-with-testable-user-stories-guinea-pigs-needed/72753/6) indicates it may have to do with google blocking the stock api that is suggested in the instructions for the project:  https://finance.google.com/finance/info?q=NASDAQ

This project may need to be updated with a new suggestion for a free stock api to use

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Updated sample project URL and added information about a proxy for stock price information.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #18023 and https://github.com/freeCodeCamp/freeCodeCamp/issues/17818

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/stock-price-checker.english.md [hunk 1]
```diff
@@ -33,7 +34,7 @@ tests:
     testString: ''
   - text: If I pass along 2 stocks, the return object will be an array with both stock's info. Instead of likes, it will display rel_likes(the difference between the likes on both stocks) on both.
     testString: ''
-  - text: 'A good way to receive current price is the following external API(replacing ''GOOG'' with your stock): https://finance.google.com/finance/info?q=NASDAQ%3aGOOG'
+  - text: 'A good way to receive current prices is through our stock price proxy (replacing ''goog'' with your stock symbol): https://repeated-alpaca.glitch.me/v1/stock/goog/quote'
```
Comment (Manish-Giri) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/stock-price-checker.english.md]:
```suggestion
  - text: 'A good way to receive current prices is through our stock price proxy (replacing ''GOOG'' with your stock symbol): https://repeated-alpaca.glitch.me/v1/stock/GOOG/quote'
```
Just wondering if we should maybe capitalize the stock name to `GOOG`, following stock name conventions.

I did see that the app description uses lowercase though - 
> symbol = msft | goog | aapl | ...

Comment (scissorsneedfoodtoo) [reply curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/stock-price-checker.english.md]: Thank you for catching that. Would be good to follow the stock name convention in the description here and in the Glitch example. I'll go ahead and make those changes now.

**Comment by scissorsneedfoodtoo** [comment]
Thought I should update the URLs for the other languages while I was at it. Also made a couple of minor changes to the test text here. If these changes are approved I'll update the Glitch project to match.

**Review by Manish-Giri — APPROVED** [review]
LGTM 💯

**Comment by scissorsneedfoodtoo** [comment]
Thank you for reviewing this @Manish-Giri. Also a big thank you to @Em-Ant for not only fixing this project, but also for building a proxy for the stock price API.

**Pull request merged**